ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -134,7 +136,28 @@ Delete Pipeline Run
     Navigate To Page    Data Science Pipelines    Runs
     Wait Until Page Contains Element    xpath://span[text()='Triggered']
     Click Element    //span[text()='Triggered']
-    Pipelines.Click Action From Actions Menu    ${pipeline_name}    Delete
+    Pipelines.Click Action From Actions Menu    ${run_name}    Delete
     Handle Deletion Confirmation Modal    ${run_name}    triggered run
     Wait Until Page Contains Element    xpath://h2[contains(text(), 'No triggered runs yet')]
-    Capture Page Screenshot
+
+Delete Pipeline
+    [Documentation]    Delete a pipeline. From the left menu select "Data Science Pipelines" -> Pipelines.
+    ...                The "Delete Pipeline" will search for a line in the grid that match the pipeline name.
+    ...                Based on that, hit the ... Menu in the row and hit Delete drop down menu.
+    [Arguments]    ${pipeline_name}
+    Navigate To Page    Data Science Pipelines    Pipelines
+    Wait Until Page Contains Element    xpath://a[text()='${pipeline_name}']
+    Pipelines.Click Action From Actions Menu    ${pipeline_name}    Delete
+    Handle Deletion Confirmation Modal    ${pipeline_name}    pipeline
+    Wait Until Page Contains Element    xpath://h4[contains(text(), 'No pipelines yet')]
+
+Delete Data Science Project
+    [Documentation]    Delete a Data Science Project. From the left menu select "Data Science Projects".
+    ...                The "Delete Data Science Project" will search for a line in the grid that match the project name.
+    ...                Based on that, hit the ... Menu in the row and hit Delete drop down menu.
+    [Arguments]    ${project_name}
+    Open Data Science Projects Home Page
+    Wait Until Page Contains Element    xpath://a[text()='${project_name}']
+    Pipelines.Click Action From Actions Menu    ${project_name}    Delete
+    Handle Deletion Confirmation Modal    ${project_name}    project
+    Wait Until Page Contains Element    xpath://h2[contains(text(), 'No data science projects yet')]
```

Comment:
can you reuse the common keyword to delete the project?  

`Delete Data Science Project` from`ODHDataScienceProject/Projects.resource`

Reply:
Done